photutils/detection/findstars.py

```diff
@@ -326,61 +326,6 @@ def __init__(self, threshold, fwhm, sigma_radius=1.5, minsep_fwhm=2.5,
         self.exclude_border = exclude_border
 
     def find_stars(self, data):
-        """
```

Comment:
I'm still confused about this commit.  The lines that you remove here (329-383) are *not* in the current master version of `findstars.py` (i.e. I already removed them):
https://github.com/astropy/photutils/blob/master/photutils/detection/findstars.py#L329